Gemfile

```diff
@@ -91,12 +65,14 @@ group :development, :test do
   gem "factory_girl_rails"
   gem "database_cleaner"
   gem "capybara-select2"
-  gem "show_me_the_cookies"
-  gem "brakeman"
+  gem 'brakeman', require: false
   gem "pry"
   gem "pry-remote"
   gem 'pry-byebug'
+  gem 'ruby-prof'
+end
+
+group :development do
```

Comment:
Custom design be icing on the cake (and they don't need to be fancy haha), but even maybe more useful information/language on them wouldn't be a terrible idea? Like `if the error condition persists, you can contact us at <SOME EMAIL ADDRESS> with additional detail, including steps to reproduce the error....blah blah`.